CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [0.5.5] - 2023-07-26
```

Comment:
```suggestion
## [0.6.0] - 2023-07-26
```

Reply:
Let's bump the minor version (not the patch version) since we are renaming the module.